scripts/check_style.sh

```diff
@@ -16,6 +16,13 @@ EXCLUDE_FILES=(
     "test/scripts/fixtures/smt_contract_with_cr_newlines.sol"
     "test/scripts/fixtures/smt_contract_with_mixed_newlines.sol"
 )
+
+# Also exclude files that are placed within a directory,
+# where its name is starting with '_' (e.g. for external sources used in semantic tests).
+for EXTERNAL_SOURCE in $(find test/libsolidity/semanticTests | grep "/_")
```

Comment:
Why not pass the name directly to `find`? Would this work?

```suggestion
for EXTERNAL_SOURCE in $(find test/libsolidity/semanticTests -name '*/_*')
```

Reply:
I reverted the changes here. If we add OZ or another repository, we can easily add a similar mechanism to exclude such files from being checked. But for this PR it is not needed.